kasa/iot/modules/ambientlight.py

```diff
@@ -14,9 +15,27 @@
 class AmbientLight(IotModule):
     """Implements ambient light controls for the motion sensor."""
 
+    def __init__(self, device, module):
+        super().__init__(device, module)
+        self._add_feature(
+            Feature(
+                device=device,
+                container=self,
+                name="Ambient Light",
+                icon="mdi:brightness-percent",
+                attribute_getter="ambientlight_brightness",
+                type=FeatureType.Sensor,
+            )
+        )
+
     def query(self):
         """Request configuration."""
-        return self.query_for_command("get_config")
+        req = merge(
+            self.query_for_command("get_realtime"),
+            self.query_for_command("get_current_brt"),
```

Comment:
Could you add an example output for these commands to the PR description? We don't currently have tests for module responses, but that may change at some point in the future.

Reply:
Sure, the response looks like:

```{'get_current_brt': {'value': 26, 'err_code': 0}```

I'll add it to the PR description.

Reply:
Added CLI output from a dimmer with ambient light sensor to the PR description as well.

Reply:
Cool, thanks! :+1: Feel free to add other features to that device (or others you may have), if there is something worth exposing similarly.